labman/db/composition.py

```diff
@@ -478,16 +479,36 @@ def _get_sample_composition_type_id(compostion_type):
             sct_id = TRN.execute_fetchlast()
         return sct_id
 
+    @staticmethod
+    def generate_content(sample_name, well):
+        """Fully qualify sample name by munging it with plate name & well id
+
+        Parameters
+        ----------
+        sample_name : str
+            The base name for the sample, such as 1.SKB1.640202 or blank
+        well: labman.db.container.Well
+            The well object in which this sample is placed
+
+        Returns
+        -------
+        A string of the format samplename.plate_name.wellcolumnletrownum
+        """
+        munged_plate_name = re.sub('\s+', '.', well.plate.external_id)
```

Comment:
Unless I'm missing something, this regex can be changed to `[\s,_]+`, and it should make the next line unnecessary (I checked with [pythex](https://pythex.org/?regex=%5B%5Cs%2C_%5D%2B&test_string=this%20is%20a%20_%20test_a_b_c_____d_E&ignorecase=0&multiline=0&dotall=0&verbose=0)).